.idea/codeStyles/Project.xml

```diff
@@ -16,7 +16,7 @@
         </value>
       </option>
     </AndroidXmlCodeStyleSettings>
-    <JSCodeStyleSettings version="0">
+    <JSCodeStyleSettings>
```

Comment:
Discard all changes in that file please

Reply:
sorry, i couldn't :( it's automatically changes now

Reply:
Remove all folder **.idea** from git

And change **.gitignore**
Delete  this block
```
# Exclude Code Style
!/.idea/codeStyles/
```
And change line
```.idea/*``` 
to 
```.idea```